src/sim/fd_array.hh

```diff
@@ -36,9 +36,9 @@
 #include <map>
 #include <memory>
 #include <string>
-
 #include "sim/fd_entry.hh"
 #include "sim/serialize.hh"
+#include "sim/sim_object.hh"
```

Comment:
Don't include this.

Instead, inside the `gem5` namespace add the line `class Process;` This is a "forward declaration"

Reply:
I added it. Now I know what is "forward declaration". Thank you!